### ISSUE TITLE
[Help]: 底模训练时长求助

### ISSUE THREAD

**Issue by FlyToYourMooN** [svc-develop-team/so-vits-svc#112]
### 请勾选下方的确认框。

- [X] 我已仔细阅读[README.md](https://github.com/svc-develop-team/so-vits-svc/blob/4.0/README_zh_CN.md)和[wiki中的Quick solution](https://github.com/svc-develop-team/so-vits-svc/wiki/Quick-solution)。
- [X] 我已通过各种搜索引擎排查问题，我要提出的问题并不常见。
- [X] 我未在使用由第三方用户提供的一键包/环境包。

### 系统平台版本号

Ubuntu

### GPU 型号

3090

### Python版本

3.9

### PyTorch版本

1.13

### sovits分支

4.0(默认)

### 数据集来源（用于判断数据集质量）

多个开源的单人歌声数据集，质量较高

### 出现问题的环节或执行的命令

训练

### 问题描述

我需要训练一个16K采样率的底模，使用了混合说话和歌声数据集，总共30000多条数据，训练到200K步，Mel loss到了在14左右，推理时有电流音。请问一下底模训练时使用了多大规模的数据，训练了多久，Mel loss到了多少呢

### 日志

```python
无
```

### 截图`so-vits-svc`、`logs/44k`文件夹并粘贴到此处

![image](https://user-images.githubusercontent.com/32425101/229339700-a5fc4493-9b71-434b-9667-0081a7aee8c7.png)

### 补充说明

hop size = 320 
upsample = 10,4,2,2,2
sigment size = 8960

**Comment by Popo-Neko** [comment]
![image](https://user-images.githubusercontent.com/126398377/229489165-3ff4e69b-db11-4eb3-bef8-91eab80dda89.png)

**Comment by leng-yue** [comment]
训练底膜和 finetune 是不一样的, 就经验而言, 一般起吗要 20-30w 步

**Comment by SuCicada** [comment]
> 训练底膜和 finetune 是不一样的, 就经验而言, 一般起吗要 20-30w 步

请问如果 loss 出现不降反升的情况，继续增加训练步数会有帮助吗

**Comment by sgorillad** [comment]
@Popo-Neko 请教下，上面这个图是用什么工具绘制的，数据源在工程中哪里哈，感谢~

**Comment by Popo-Neko** [comment]
> 训练底膜和 finetune 是不一样的, 就经验而言, 一般起吗要 20-30w 步

原来是这样啊，一开始的G_0.pth和D_0.pth是底模，然后其实我是在这个基础上再继续训练，我搞错了

**Comment by Popo-Neko** [comment]
> @Popo-Neko 请教下，上面这个图是用什么工具绘制的，数据源在工程中哪里哈，感谢~

这个是tensorboard，你查查你的深度学习框架支不支持它，tensorflow和pytorch里面都有，后面那句没理解什么意思，数据源要自己找然后来训练吧

**Comment by FlyToYourMooN** [comment]
> 训练底膜和 finetune 是不一样的, 就经验而言, 一般起吗要 20-30w 步

我的Batch size为32，训练了200K步，效果仍然不理想

**Comment by leng-yue** [comment]
你得放点输入和输出的 sample 上来

**Comment by phigrey** [comment]
@Popo-Neko 请问G_0.pth 和 D_0.pth 可以分享下吗？innky 那个项目的pretrained 被删了。
